### ISSUE TITLE
♿Accordian AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1317]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
[Ensure Windows high contrast mode is supported.
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1180.html)
> Chevrons are invisible in high contrast mode on Windows 10
![capture](https://user-images.githubusercontent.com/40970507/47396686-77ca2900-d6f1-11e8-8075-049a1c266d30.PNG)

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

**Comment by joshblack** [comment]
@dakahn does the landmark/region part apply here? Rarely would this be a landmark on a page, I would think, it would always be a part of one.

**Comment by dakahn** [comment]
Totally right, meant to get rid of that. I'll fix it.

**Comment by dakahn** [comment]
closing -- this was fixed upstream by carbon-elements https://github.com/IBM/carbon-elements/pull/388
